prompt2model/dataset_retriever/column_selection_prompt.py

```diff
@@ -55,17 +94,25 @@ def truncate_row(example_row: dict, max_length=50) -> str:
     """Truncate the row before displaying if it is too long."""
     truncated_row = {}
     for key in example_row.keys():
-        truncated_row[key] = json.dumps(example_row[key])[:max_length] + "..."
+        curr_row = json.dumps(example_row[key])
+        truncated_row[key] = (
+            curr_row if len(curr_row) <= max_length else curr_row[:max_length] + "..."
```

Comment:
nit - Might want to do `len(curr_row) <= max_length - 3` if you want the truncated row with "..." to have a consistent length with the non-truncated rows